_components/button/guidance/implementation.md

```diff
@@ -1,2 +1,3 @@
 - The examples demonstrate how to use button elements. To use a button style on a link, add the `usa-button` class to your link.
-- To use a different style button on your link, add button variant classes in addition to `usa-button`.
\ No newline at end of file
+- To use a different style button on your link, add button variant classes in addition to `usa-button`.
+- Use the browser-native `disabled ` attribute for any disabled button. Don't use  `usa-button--disabled`, which is intended only for debugging and prototyping.
```

Comment:
```suggestion
- Use the browser-native `disabled` attribute for any disabled button. Don't use  `usa-button--disabled`, which is intended only for debugging and prototyping.
```


Currently looks like this with extra space:
![image](https://user-images.githubusercontent.com/3385219/119874227-00815500-beeb-11eb-8ebc-87b13fdecb14.png)